src/Tribe/Aggregator/Service.php

```diff
@@ -172,7 +172,14 @@ public function get( $endpoint, $data = array() ) {
 			return $response;
 		}
 
-		if ( isset( $response->data ) && isset( $response->data->status ) && '404' === $response->data->status ) {
+		if (
+			(
+				isset( $response->data )
+				&& isset( $response->data->status )
+				&& '404' === $response->data->status
+			)
+		     || '200' != wp_remote_retrieve_response_code( $response )
```

Comment:
I wonder if we should break the `200` out into its own if block with its own error message? And perhaps it would be a good plan to have a separate error message specifically for the `403` we are sending when we throttle.